src/routes/v1/index.js

```diff
@@ -0,0 +1,11 @@
+import {Router} from 'express';
```

Comment:
I think we should consider using prettier or standard to get some of these things as usual, you'd usually put a space before and after `Router`